plugins/draw-tools.js

```diff
@@ -584,8 +584,8 @@ window.plugin.drawTools.snapToPortals = function() {
       if (visibleBounds.contains(ll)) {
         testCount++;
         var newll = findClosestPortalLatLng(ll);
```

Comment:
```suggestion
        var newll = findClosestPortalLatLng(ll);
        // Using a hard comparison instead of a soft "equal" to fix "snap to portals" precision.
        // See 
https://github.com/IITC-CE/ingress-intel-total-conversion/pull/481
```

Reply:
I suppose that comments would be redundant here.
And in case of need - more info is available in commit message.

Reply:
At the very least, we need a link to PR/commit. Otherwise, during refactoring we may accidentally return the past state of code